app/index.html

```diff
@@ -150,6 +150,58 @@
     </div>
   </ui-minimize-panel>
 
+  <ui-minimize-panel title="File list" open='false'>
+    <div class='panel panel-default'>
+      <div class='panel-heading'>
+        File List
+      </div>
+      <div class='panel-body' ng-controller="FileListController">
+        <div ng-if="file_list.files.length > 0">
+          <input type="checkbox"
+                 name="checked_files[]"
+                 value="Select all"
+                 ng-checked="all_selected"
+                 ng-click="select_all()">
+          Select all
+          <table>
+          <tr>
+            <th>Filename</th>
+            <th>Size</th>
+            <th>Last modified</th>
+            <th>Tags</th>
+          </tr>
+          <tr ng-repeat="file in file_list.files">
+            <td>
+              <input type="checkbox"
+                   name="checked_files[]"
+                   ng-checked="selected.indexOf(file.id) > -1"
+                   ng-click="toggle_file(file.id)">
+              <a href="#{{ $routeSegment.getSegmentUrl('preview', {id: file.id}) }}">{{ file.title }}</a>
```

Comment:
Yeah, I'm planning in a later for file info to be fetched via the `File` service, so it still works even when the list is empty.